### ISSUE TITLE
Warning message abbreviates path string; warning is less useful

### ISSUE THREAD

**Issue by jtmoon79** [pypa/pipenv#3521]
**tl;dr** pipenv emitted a warning but shortened the path to the particular virtual environment it was referring `(C:\Users\user1\.\J\S\python.exe)`. It was difficult to act on the warning because I am not certain which virtual environment (among many I have setup) or which python install (several are installed) this refers (I initially thought pipenv mixed up paths).

### Issue description

I have setup five pipenv virtual environments for projects I am hacking at.  Also is installed Pythons 3.4, 3.5, 3.6, 3.7.

When I setup another pipenv virtual environment, pipenv warned me about a possible problem.

```
Virtualenv location: C:\Users\user1\.virtualenvs\Project1-u18NbQNd
Warning: Your Pipfile requires python_version 3.7, but you are using 3.5.4 (C:\Users\user1\.\J\S\python.exe).
  $ pipenv --rm and rebuilding the virtual environment may resolve the issue.
  $ pipenv check will surely fail.
```

I cannot determine which project virtual environment the shortened path "`C:\Users\user1\.\J\S\python.exe`" refers.  And I'm not certain if that path is simply the same as the preceding sentence path "`C:\Users\user1\.virtualenvs\Project1-u18NbQNd`" (I would guess it is, but I'd prefer to be certain).  (it turns out the `python_version` within `Pipfile` was wrong)

Since I have several Python installs (3.4, 3.5, 3.6, 3.7) and several pipenv virtual environments, I was concerned pipenv was using the wrong pathed Python environment.

### Recommended Fix

Print the entire path within the warning message.

**Comment by techalchemy** [comment]
Hey, thanks for the report and sorry for the issue -- it can only possibly refer to the one you are working on (`pipenv --venv` etc)

I do see why the shortened path might be confusing, but it also provides consistency for the UX especially for terminal users.  In any event this is definitely low priority so I think for now we can close this out and revisit in the future if it continues to cause confusion
